client/src/App.js

```diff
@@ -12,6 +13,9 @@ function App() {
     <MuiThemeProvider theme={theme}>
       <BrowserRouter>
         <Route path="/" component={LandingPage} />
```

Comment:
You can remove this route to the boilerplate component

Reply:
Done